integration_tests/db/queries.ts

```diff
@@ -301,6 +310,20 @@ async function updateRiskProfile({
   return await db.query(`update form set risk_profile = $1::JSON where booking_id = $2`, [riskProfile, bookingId])
 }
 
+const updateFormRecord = async ({ bookingId, status, formResponse }: { bookingId: number, status: string, formResponse: any }) => {
+  const existingFormResponse = await db.query(`select form_response from form where booking_id = $1`, [bookingId])
+  return await db.query(
+    `update form set status = $1, form_response = $2 where booking_id = $3`,
+    [status, JSON.stringify({ ...existingFormResponse.rows[0]?.form_response, ...formResponse }), bookingId]
```

Comment:
probably not a concern here - not sure why ramda mergeRight is imported above, but could maybe also be used here?

the outcome of merging two json objects is not a deep merge, but a replacement at the key level. Ramda mergeRight is probably a deep merge - the intuitive outcome. From what I can see of usage here below, it's fine as is.